src/components/FrameworkLinks.tsx

```diff
@@ -0,0 +1,131 @@
+import { Bun as IconBun } from "@/components/icons/tech/Bun";
+import { Deno as IconDeno } from "@/components/icons/tech/Deno";
+import { NestJs as IconNestJs } from "@/components/icons/tech/NestJs";
+import { NextJs as IconNextJs } from "@/components/icons/tech/NextJs";
+import { NodeJs as IconNodeJs } from "@/components/icons/tech/NodeJs";
+import { Remix as IconRemix } from "@/components/icons/tech/Remix";
+import { SvelteKit as IconSvelteKit } from "@/components/icons/tech/SvelteKit";
+
+import Button from "@/components/Button";
+import { queryParamFramework } from "@/store";
+import { useStore } from "@nanostores/react";
+import type { ForwardedRef, PropsWithChildren } from "react";
+import { forwardRef, useEffect, useState } from "react";
+
+import styles from "./FrameworkLinks.module.scss";
+
+/**
+ * Framework Links
+ *
+ * Renders a list of buttons that switch to a specific framework.
+ */
+const FrameworkLinks = forwardRef(
+  ({ ...props }: PropsWithChildren, ref: ForwardedRef<HTMLDivElement>) => {
+    const [hide, setHide] = useState(false);
```

Comment:
I think the initial state here needs to check the params because otherwise I get a flash of content when changing the browser URL.

Reply:
Can't check the URL on the server but I can default to hidden instead [a3b3d42](https://github.com/arcjet/arcjet-docs/pull/230/commits/a3b3d4214d356b96a3b2ade833f1925303581c1e).